tftrt/examples/image-classification/image_classification.py

```diff
@@ -562,7 +560,7 @@ def get_frozen_graph(
             outputs=['logits', 'classes'],
             max_batch_size=batch_size,
             max_workspace_size_bytes=max_workspace_size,
-            precision_mode=precision,
```

Comment:
Here this line can cause the following error when run example `python image_classification.py --model resnet_v1_50 \
    --data_dir /data/imagenet/train-val-tfrecord \
    --use_trt \
    --precision fp16`

`, line 569, in __init__
    TrtPrecisionMode.supported_precision_modes))
ValueError: precision mode 'fp16' is not supported.It should be one of <function supported_precision_modes at 0x7fdf2415b6e0>
`

Reply:
Could you run this example with the latest tensorflow and tensorrt?

Reply:
I see. It's not because of latest TRT. In the latest TF, we dropped support of lowercase. Thanks for the PR.

Reply:
I see

Reply:
There are other changes in this PR which we don't want. We only want this single line that does `upper()`. Could you remove the other changes.